tests/wallet/dao_wallet/test_dao_wallets.py

```diff
@@ -810,7 +812,7 @@ async def get_dao_rules(wallet):
 
     # Give the wallet nodes a second
     await asyncio.sleep(1)
-    for i in range(1, 12):
+    for i in range(1, 20):
```

Comment:
This is A LOT of blocks, this test will likely be very slow.  Again, I think you should check out some of the methods on the full node simulator, they're useful for waiting to progress in a test.

Reply:
brought it down now